SpywareFilter/sections/general_extensions.txt

```diff
@@ -4,7 +4,7 @@
 !
 !! facebook.com - removing tracking parameter
 !! external links
-facebook.com,facebookcorewwwi.onion#%#new MutationObserver(function(){document.querySelectorAll('a[target="_blank"]').forEach(function(t){if(t.href.contains("?fbclid")||t.href.contains("%3Ffbclid")||t.href.contains("&fbclid")||t.href.contains("%26fbclid")||t.href.contains("%3Futm")||t.href.contains("?utm")||t.href.contains("&fbc=")||t.href.contains("%26fbc%3D")||t.href.contains("?share=")||t.href.contains("%3Fshare%3D")){var e=t.href.split("#!")[1];if(!e)var e=t.href.split("%23%21")[1];if(!e)var e="";t.href.contains("#!")&&(e="#!"+e),t.href.contains("%23%21")&&(e="%23%21"+e),t.setAttribute("href",t.href.split("?fbclid")[0]+e),t.setAttribute("href",t.href.split("%3Ffbclid")[0]+e),t.setAttribute("href",t.href.split("&fbclid")[0]+e),t.setAttribute("href",t.href.split("%26fbclid")[0]+e),t.setAttribute("href",t.href.split("%3Futm")[0]+e),t.setAttribute("href",t.href.split("?utm")[0]+e),t.setAttribute("href",t.href.split("&fbc=")[0]+e),t.setAttribute("href",t.href.split("%26fbc%3D")[0]+e),t.setAttribute("href",t.href.split("?share=")[0]+e),t.setAttribute("href",t.href.split("%3Fshare%3D")[0]+e)}})}).observe(document,{childList:!0,subtree:!0});
+facebook.com,facebookcorewwwi.onion#%#new MutationObserver(function(){document.querySelectorAll('a[target="_blank"]').forEach(function(t){if(t.href.contains("?fbclid")||t.href.contains("%3Ffbclid")||t.href.contains("&fbclid")||t.href.contains("%26fbclid")||t.href.contains("%3Futm")||t.href.contains("?utm")||t.href.contains("&fbc=")||t.href.contains("%26fbc%3D")||t.href.contains("?share=")||t.href.contains("%3Fshare%3D")||t.href.contains("%3F__twitter_impression%3D")||t.href.contains("?__twitter_impression=")){var e=t.href.split("#!")[1];if(!e)var e=t.href.split("%23%21")[1];if(!e)var e="";t.href.contains("#!")&&(e="#!"+e),t.href.contains("%23%21")&&(e="%23%21"+e),t.setAttribute("href",t.href.split("?fbclid")[0]+e),t.setAttribute("href",t.href.split("%3Ffbclid")[0]+e),t.setAttribute("href",t.href.split("&fbclid")[0]+e),t.setAttribute("href",t.href.split("%26fbclid")[0]+e),t.setAttribute("href",t.href.split("%3Futm")[0]+e),t.setAttribute("href",t.href.split("?utm")[0]+e),t.setAttribute("href",t.href.split("&fbc=")[0]+e),t.setAttribute("href",t.href.split("%26fbc%3D")[0]+e),t.setAttribute("href",t.href.split("?share=")[0]+e),t.setAttribute("href",t.href.split("%3Fshare%3D")[0]+e),t.setAttribute("href",t.href.split("%3F__twitter_impression%3D")[0]+e),t.setAttribute("href",t.href.split("?__twitter_impression=")[0]+e)}})}).observe(document,{childList:!0,subtree:!0});

```

Comment:
why not just `true` instead of `!0`? 
Maybe `characterData: true` will not be superfluous.

Reply:
All the other observe rules we have in filters have !0